fixup-mountpoints

```diff
@@ -143,7 +143,37 @@ case "$DEVICE" in
             -e 's block/platform/msm_sdcc.1/by-name/modem mmcblk0p1 ' \
```

Comment:
Please rename this commit as per others in this project: "[codename] Mountpoints for FULL_DEVICE_NAME"